test/core/test_parser.py

```diff
@@ -98,6 +98,18 @@ def test_load_templates(example_test_filesystem):
     assert len(templates) > 0
 
 
+def test_load_templates(example_test_filesystem):
```

Comment:
we may have flipped the function name. this called `load_template` but function name is `test_load_templates`. The previous test is testing `load_templates` but function name is `test_load_template`. The two test case is swapped.

Reply:
Nice! I'll swap them.